### PR TITLE
bump(main/libassuan): 3.0.1, bump(x11/pinentry-gtk): 1.3.1, revbump(revdeps)

### PR DESCRIPTION
**Pull request by twaik**
Warning. Partial upgrade breaks `apt`/`pkg`, required to install `gnupg` package too.
Pinentry is broken too.
<details>

```
~ $ pkg up
Checking availability of current mirror:
[*] https://packages-cf.termux.dev/apt/termux-main/: ok
Get:1 https://packages-cf.termux.dev/apt/termux-main stable InRelease [14.0 kB]
Get:2 https://packages-cf.termux.dev/apt/termux-x11 x11 InRelease [14.0 kB]
Err:1 https://packages-cf.termux.dev/apt/termux-main stable InRelease
  At least one invalid signature was encountered.
Err:2 https://packages-cf.termux.dev/apt/termux-x11 x11 InRelease
  At least one invalid signature was encountered.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
W: https://packages-cf.termux.dev/apt/termux-main/dists/stable/InRelease: Unknown response from gpgv to --assert-pubkey-algo check: CANNOT LINK EXECUTABLE "gpgv": cannot locate symbol "assuan_release" referenced by "/data/data/com.termux/files/usr/bin/gpgv"...
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://packages-cf.termux.dev/apt/termux-main stable InRelease: At least one invalid signature was encountered.
W: https://packages-cf.termux.dev/apt/termux-x11/dists/x11/InRelease: Unknown response from gpgv to --assert-pubkey-algo check: CANNOT LINK EXECUTABLE "gpgv": cannot locate symbol "assuan_release" referenced by "/data/data/com.termux/files/usr/bin/gpgv"...
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://packages-cf.termux.dev/apt/termux-x11 x11 InRelease: At least one invalid signature was encountered.
W: Failed to fetch https://packages-cf.termux.dev/apt/termux-main/dists/stable/InRelease  At least one invalid signature was encountered.
W: Failed to fetch https://packages-cf.termux.dev/apt/termux-x11/dists/x11/InRelease  At least one invalid signature was encountered.
W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
~ $ pinentry
CANNOT LINK EXECUTABLE "pinentry": cannot locate symbol "assuan_write_line" referenced by "/data/data/com.termux/files/usr/bin/pinentry-curses"...
```
</details>
Since this is major update with ABI breakage I revbump all reverse dependencies.

Needs maintainers approve.

### CODE REVIEW

**Comment by twaik** [comment]
@sylirre @Grimler91 you commited to libassuan, so I hope you will take a look.
Tests on device (pinentry, apt) are fine after installing revbumped pinentry and gnupg+gpgv so I revbumped all reverse dependencies.

**Comment by twaik** [comment]
So in this case the packages listed in TERMUX_PKG_BREAKS will be automatically updated even if their update was not triggered by user, right?

**Comment by twaik** [comment]
Will `TERMUX_PKG_BREAKS="gnupg (< 2.4.5-3)"` be fine (without `=`)?

**Comment by twaik** [comment]
Done.

File: packages/libassuan/build.sh [hunk 1]
```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.gnupg.org/related_software/libassuan/
...
-TERMUX_PKG_VERSION="2.5.7"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="3.0.1"
 TERMUX_PKG_SRCURL=https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-${TERMUX_PKG_VERSION}.tar.bz2
-TERMUX_PKG_SHA256=0103081ffc27838a2e50479153ca105e873d3d65d8a9593282e9c94c7e6afb76
+TERMUX_PKG_SHA256=c8f0f42e6103dea4b1a6a483cb556654e97302c7465308f58363778f95f194b1
 TERMUX_PKG_DEPENDS="libgpg-error"
 TERMUX_PKG_BREAKS="libassuan-dev"
 TERMUX_PKG_REPLACES="libassuan-dev"
+TERMUX_PKG_BREAKS="gnupg (<< 2.4.5-3), gpgv (<< 2.4.5-3), gpgme (2.4.5-3), pinentry (<< 1.3.1-1), pinentry-gtk (<< 1.3.1), profanity (<< 0.14.0-3)"
```
Comment (truboxl) [inline packages/libassuan/build.sh:11]: Should be on the same line with the other BREAKS. No operator for gpgme?

Comment (twaik) [reply packages/libassuan/build.sh]: Sorry, missed this one. Done.

**Comment by twaik** [comment]
Seems fine to me now (since it does not allow breaking environment). It will be merged to master on Sunday if no one minds.

**Comment by Biswa96** [comment]
I can not update packages.

```
$ apt upgrade
The following packages have been kept back:
  libassuan
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.

**Comment by truboxl** [comment]
gpgme break version set incorrectly

**Comment by twaik** [comment]
Sorry. That is completely my fault. Probably copy-paste issue. But I do not understand why it did not show any error in local tests.

**Comment by Biswa96** [comment]
> I can not update packages. Is that expected?

Answering my own question. `apt reinstall libassuan` fixed the issue.

**Pull request merged**